lib/src/leak_tracker.dart

```diff
@@ -24,11 +24,16 @@ ObjectTracker _theObjectTracker() {
 ///
 /// The leak tracking will function only for debug/profile/developer mode.
 /// See usage guidance at https://github.com/dart-lang/leak_tracker.
-void enableLeakTracking({LeakTrackingConfiguration? config}) {
+void enableLeakTracking({
+  LeakTrackingConfiguration? config,
+  bool resetIfEnabled = false,
```

Comment:
This name is a bit confusing, I don't get an idea of what is accomplished when it is set.

Reply:
Renamed to resetIfAlreadyEnabled. Does it help?